testing/testcafe/tests/navigation/pivotGrid/data.ts

```diff
@@ -0,0 +1,4489 @@
+/* eslint-disable spellcheck/spell-checker */
```

Comment:
Just changed js -> ts and reformat the file with the eslint ts rules. 
(Because the testcafe starts to fail with an error at the *.js import) 

The data of the file wasn't changed.

Reply:
It's not actual anymore. 
The problem with importing *.js is gone after rebasing.
Therefore I roll back this change.